VultisigApp/VultisigApp/Services/Solana/SolanaService.swift

```diff
@@ -211,29 +162,22 @@ class SolanaService {
         let fees = response.result.map { $0.prioritizationFee }
         let nonZeroFees = fees.filter { $0 > 0 }
         
-        // Calculate the high priority fee
         let highPriorityFee = nonZeroFees.max() ?? 0
-        
         return UInt64(highPriorityFee)
-        
     }
     
     private func postRequest(with body: [String: Any]) async throws -> Data {
-        
         var request = URLRequest(url: rpcURL)
-        
+        request.cachePolicy = .returnCacheDataElseLoad
```

Comment:
It should be configurable for specific request. Otherwise it can return outdated data for state specific data, like swap quote for ex

Reply:
Or we want to cache all solana rpc requests?

Reply:
We want to cache them all for 2 min only so we don't reach the request limits.